lib/DDG/Spice/DogoBooks.pm

```diff
@@ -2,12 +2,15 @@ package DDG::Spice::DogoBooks;
 
 use DDG::Spice;
 
-triggers any => "dogobooks", "dogo books", "dogo", "kids", "kid", "child", "children";
+triggers startend => "dogobooks", "dogo books", "dogo";
+triggers start => "kids", "kid", "child", "children";
+triggers end => "books";
 
 spice to => 'http://api.dogomedia.com/api/v2/books/search.json?query=$1&api_key={{ENV{DDG_SPICE_DOGO_APIKEY}}}';
 spice wrap_jsonp_callback => 1;
 
 handle query_lc => sub {
+    
     return $_ if $_ =~ /book/i;
```

Comment:
We should also verify it matches "kid(s)/children(s) book(s)" here or "book(s) for kids/children" to prevent it from triggering on queries like "Harry Potter and the Cursed Child book"